ci/infra/testrunner/skuba/skuba.py

```diff
@@ -170,6 +170,21 @@ def _run_skuba(self, cmd, cwd=None):
         if cwd is None:
             cwd = self.cwd
 
+        if verbosity is None:
+           verbosity = self.conf.skuba.verbosity  # noqa: E111
```

Comment:
indent is off, need one more space

Reply:
I thought it looked better with the variables vertically aligned, feels messed up with that weird one-char offset. Python needs to support `iff` :)

Reply:
Extra space added so it's wonky like the preceding block. ;)